source/computer/commands.rst

```diff
@@ -63,54 +62,495 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
 
     -   其他
 
+        - `df`_
+        - `du`_
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
+        - `top`_
 
 ----
 
 cat
 ---
 
+``cat`` 命令名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来::
+
+    # 将一个文件的内容输出到终端
+    $ cat file
+    
+    # 同时显示行号
+    $ cat -n file
+    
+    # 将文件 file1 和 file2 合并到一个文件 target_file 中
+    $ cat file1 file2 > target_file
+
+diff
+----
+
+``diff`` 命令的命名来自 **diff**\ erence。该命令可以用来逐行比较文件的异同::
+
+    $ diff file1 file2
```

Comment:
Should we add a simple example showing and explaining output of diff.

Reply:
Check the example in f0a51a2.